src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -0,0 +1,151 @@
+                   help='The name of the transform.')
+        c.argument('job_name', name_arg_type, id_part='child_name_2',
+                   help='The name of the job.')
+
+    with self.argument_context('ams job list') as c:
+        c.argument('account_name', id_part=None)
+
+    with self.argument_context('ams job start') as c:
+        c.argument('priority', arg_type=get_enum_type(Priority),
+                   help='The priority with which the job should be processed.')
+        c.argument('description', help='The job description.')
+        c.argument('input_asset_name',
+                   arg_group='Asset Job Input',
+                   help='The name of the input asset.')
+        c.argument('output_asset_names',
+                   nargs='+', help='Space-separated list of output asset names.')
+        c.argument('base_uri',
+                   arg_group='Http Job Input',
+                   help='Base uri for http job input. It will be concatenated with provided file names. If no base uri is given, then the provided file list is assumed to be fully qualified uris.')
+        c.argument('files',
+                   nargs='+',
+                   help='Space-separated list of files. It can be used to tell the service to only use the files specified from the input asset.')
+
+    with self.argument_context('ams job cancel') as c:
+        c.argument('delete', help='Delete the job being cancelled.')
+
+    with self.argument_context('ams streaming') as c:
+        c.argument('account_name', account_name_arg_type, id_part='name',
+                   help='The name of the Azure Media Services account within the resource group.')
+        c.argument('default_content_key_policy_name',
+                   help='The default content key policy name used by the streaming locator.')
+
+    with self.argument_context('ams streaming locator') as c:
+        c.argument('streaming_locator_name', name_arg_type, id_part='child_name_1',
+                   help='The name of the streaming locator.')
+        c.argument('asset_name',
+                   help='The name of the asset used by the streaming locator.')
+        c.argument('streaming_policy_name',
+                   help='The name of the streaming policy used by the streaming locator.')
+        c.argument('start_time',
+                   help="Start time (Y-m-d'T'H:M'Z') of the streaming locator.")
+        c.argument('end_time',
+                   help="End time (Y-m-d'T'H:M'Z') of the streaming locator.")
+
+    with self.argument_context('ams streaming policy') as c:
+        c.argument('streaming_policy_name', name_arg_type, id_part='child_name_1',
+                   help='The name of the streaming policy.')
+        c.argument('download',
+                   arg_group='Encryption Protocols',
+                   help='Enable Download protocol.')
+        c.argument('dash',
+                   arg_group='Encryption Protocols',
+                   help='Enable Dash protocol.')
```

Comment:
Is this a Boolean property?  If you, use three_state_flag here.  https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags

Same comment for other flags below.

Reply:
Done